src/source/remote.js

```diff
@@ -46,7 +45,7 @@ class RemoteSource extends BaseSource {
     }
 
     // otherwise make a single request for each slice
-    return await Promise.all(
+    return Promise.all(
```

Comment:
It may not be obvious without expanding to get additional context, but this is the return of an `async` function.  So returning a promise is the same as returning the resolved value.